config/typescript/tsconfig.base.json

```diff
@@ -16,6 +16,7 @@
     "noUnusedParameters": true,
     "sourceMap": false,
     "noEmitOnError": false,
+    "skipLibCheck": true,
```

Comment:
This is needed to bypass the type errors as a result of pulling in different versions of React. Unfortunately argo-admin-cli still relies on Polaris which only supports React v16